packages/config/config/devices/0x0086/dsb09.json

```diff
@@ -36,7 +37,6 @@
 		"1": {
 			"label": "Voltage",
 			"description": "Voltage to calculate wattage from amperage.",
-			"unit": "V",
```

Comment:
That doesn't look correct. (same for the other removed units in this file)

Reply:
This is being done by your normalize routine, so I assumed you wanted it.